modules/ROOT/pages/apikit-graphql-api-implementation.adoc

```diff
@@ -0,0 +1,525 @@
+           "comments": [
+               {
+                   "user": {
+                       "username": "cwalesby1p"
+                   },
+                   "body": "Praesent blandit. Nam nulla. Integer pede justo, lacinia eget, tincidunt eget, tempus vel, pede.\n\nMorbi porttitor lorem id ligula. Suspendisse ornare consequat lectus. In est risus, auctor sed, tristique in, tempus sit amet, sem.",
+                   "replies": []
+               },
+               {
+                   "user": {
+                       "username": "rricci2d"
+                   },
+                   "body": "Maecenas leo odio, condimentum id, luctus nec, molestie sed, justo. Pellentesque viverra pede ac diam. Cras pellentesque volutpat dui.",
+                   "replies": []
+               }
+           ]
+       }
+   }
+}
+----
+
+== Request Mutations Examples
+
+The type *Mutation* defines the list of mutations that the service enables.
+
+----
+...
+type Mutation {
+    # Creates a new user
+    createUser(user: UserInput!): User!
+    # Creates a new post
+    createPost(userId: ID!, title: String!, body: String!): Post!
+    # Adds a comment to a post
+    postComment(userId: ID!, postId: ID!, body: String!): Comment!
+    # Adds a reply to a comment
+    replyToComment(commentId: ID!, userId: ID!, body: String!): Comment!
+}
+...
+----
+
+=== Create a New Post
+
+This example request creates a new post and asks for some information on it.
+
+==== Make a Query
+
+----
+mutation CreatePost($userId: ID!, $title: String!, $body: String!) {
+   createPost(userId: $userId,title: $title,body: $body) { 
+       id
+       user {
+           username
+       }
+       title
+       body
+       comments {
+           body
+       }
+   }
+}
+----
+
+==== Make Variables
+
+----
+{
+   "userId": 42,
+   "title": "GraphQL by example",
+   "body": "Let's talk about GraphQL"
+}
+----
+
+==== Trigger a Request
+
+----
+curl --location --request POST 'http://localhost:8081/graphql' \
+--header 'Content-Type: application/json' \
+--data-raw '{"query":"mutation CreatePost($userId:ID!,$title:String!,$body:String!){createPost(userId:$userId,title:$title,body:$body){id user{username}title body comments{body}}}","variables":{"userId":42,"title":"GraphQL by example","body":"Let'\''s talk about GraphQL"}}'
+----
+
+==== Get a Response
+
+----
+{
+   "data": {
+       "createPost": {
+           "id": "101",
+           "user": {
+               "username": "tblackmore15"
+           },
+           "title": "GraphQL by example",
+           "body": "Let's talk about GraphQL",
+           "comments": []
+       }
+   }
+}
+----
+
+=== Add a Comment to the New Post
+
+This use case posts a comment in reply to the post created in the previous step. It also asks for the service to return the body of the comment created and the original post with its list of comments to check that the new one is included.
+
+==== Make a Query
```

Comment:
I guess Make is correct :>

Reply:
Should these `====` have intro sentences?

Reply:
Regarding the "Make a Query" titles, I'll keep them. If my knowledge gives me the possibility to find a better option, I'll change them in the future.
Regarding the ==== intro sentences, it was hard for us to think a sentence that would not repeat the words of the title, so we do without them to avoid repetitions.
Thanks!